dashboard/src/components/layout/SettingsSidebar/SettingsSidebar.tsx

```diff
@@ -220,6 +226,20 @@ export default function SettingsSidebar({
             </SettingsNavLink>
           </List>
         </nav>
+        <Box className="border-t">
+          <SettingsNavLink
+            href="/editor"
+            exact={false}
+            onClick={handleSelect}
+            className="flex w-full border group-focus-within:pr-9 group-hover:pr-9 group-active:pr-9"
+            textClassName="flex w-full justify-center"
+          >
+            <div className="flex w-full flex-row items-center justify-center space-x-4 py-2.5">
+              <TerminalIcon />
```

Comment:
we should change the icon here to [this one](https://lucide.dev/icons/sliders-horizontal)